### PR TITLE
Ballot list - Closes #48

### LINKED ISSUE #48: implement getters to get ballot feed info from Ethereum and IPFS
From IPFS:
- [x] title
- [ ] author name
- [x] date of creation

From Ethereum: 
- [x] number of `yes` votes on ballot (option1)
- [x] number of `no` votes on ballot (option2)

Contextual information that we need to retrieve based on the users' eth address: 
- [ ] number votes on ballot + selected option
- [ ] available delegated and not cast votes on ballots

### PR DESCRIPTION
**Pull request by LucasIsasmendi**
Work in Progress issue #48

### CODE REVIEW

File: contracts/LiquidDemocracy.sol [hunk 1]
```diff
@@ -122,6 +122,20 @@ contract LiquidDemocracy {
         );
     }
 
+    /**
+    * @notice Gets ballot by position in array
+    */
+    function getBallot(uint position) public view returns (bytes32, address, uint, uint) {
+        require(position < ballotData.number);
+        require(0 < position + 1);
```
Comment (aecc) [inline contracts/LiquidDemocracy.sol]: position >= 0 ?

File: dApp-Svr-Exp/modules/ballots/ballots-list-ctrl.js [hunk 2]
```diff
@@ -175,38 +141,42 @@
...
-            $log.log('successfully stored', file.hash);
-            const validETHHash = apiIPFS.getBytes32FromIpfsHash(file.hash)
-            $log.log("ETH byte32: " + validETHHash);
-            apiETH.instance.createNewBallot(validETHHash, {gas: 1000000}).then(function(result) {
-              $log.log('New Ballot created:', JSON.stringify(ballot));
+            $log.log('[IPFS] - successfully stored', file.hash);
+            const ipfsHashEncoded = apiIPFS.getBytes32FromIpfsHash(file.hash)
+            $log.log("ETH byte32: " + ipfsHashEncoded);
+            apiETH.instance.createNewBallot(ipfsHashEncoded, {gas: 1000000}).then(function(result) {
+              $log.log('[ETH] - New Ballot created:', JSON.stringify(ballot));
```
Comment (aecc) [inline dApp-Svr-Exp/modules/ballots/ballots-list-ctrl.js]: +1 for these better logging

File: dApp-Svr-Exp/modules/ballots/detail/ballot-detail-ctrl.js [hunk 3]
```diff
@@ -18,6 +18,57 @@
...
+            const ipfsBallotTitle = ballot[0];
+            const owner = ballot[1];
+            vm.ballot.option1 = ballot[2].toNumber();
+            vm.ballot.option2 = ballot[3].toNumber();
+            vm.ballot.total = vm.ballot.option1 + vm.ballot.option2;
+            $log.log('Ballot: ', vm.ballot);
+            $log.log('Ballot IPFS address: ' + ipfsBallotTitle);
+            calculateTotals();
+            $scope.$apply();
+            // const validIpfsAddress = apiIPFS.getIpfsHashFromBytes32(ipfsBallotTitle);
```
Comment (aecc) [inline dApp-Svr-Exp/modules/ballots/detail/ballot-detail-ctrl.js]: should we remove this?

File: dApp-Svr-Exp/modules/ballots/ballots-list-ctrl.js [hunk 4]
```diff
@@ -82,15 +43,17 @@
       function getOption(votes, pos) {
         const totalVoted = getTotalVotes(votes);
         const desc = votes.options[pos-1];
-        const total = getPorcent(totalVoted, votes.totals[pos-1]);
```
Comment (aecc) [inline dApp-Svr-Exp/modules/ballots/ballots-list-ctrl.js]: Typos from here on: Percentage, not porcent. Can we fix them now before it spreads more?

**Comment by LucasIsasmendi** [comment]
All fixes applied. Regarding "votes bars look broken", the current UI implementation is not ready for 0 values, so we will move to a sovereign web app look and feel regarding totals

**Comment by aecc** [comment]
Now it works as a charm! LGTM

**Pull request merged**